modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -224,49 +256,89 @@ export default {
           return;
         }
 
-        this.doc.data = {
+        const body = {
           ...this.doc.data,
           ...this.docUtilityFields.data,
           ...this.docOtherFields.data
         };
         let route;
         let requestMethod;
         if (this.docId) {
-          route = `${this.moduleOptions.action}/${this.docId}`;
+          route = `${this.moduleAction}/${this.docId}`;
           requestMethod = apos.http.put;
         } else {
-          route = this.moduleOptions.action;
+          route = this.moduleAction;
           requestMethod = apos.http.post;
+
+          if (this.moduleName === '@apostrophecms/page') {
+            body._targetId = apos.page.page._id;
+            body._position = 'lastChild';
+          }
         }
 
         await requestMethod(route, {
           busy: true,
-          body: this.doc.data
+          body
         });
         this.$emit('saved');
         this.modal.showModal = false;
       });
     },
-    async newInstance () {
-      const newInstance = await apos.http.post(this.moduleOptions.action, {
-        body: {
+    async getNewInstance () {
+      try {
+        const body = {
           _newInstance: true
+        };
+
+        if (this.moduleName === '@apostrophecms/page') {
+          body._targetId = apos.page.page._id;
+          body._position = 'lastChild';
         }
-      });
+        const newDoc = await apos.http.post(this.moduleAction, {
+          body
+        });
+
+        return newDoc;
+      } catch (error) {
+        await apos.notify('Error while creating a new, empty item. Please check configured content types.', {
```

Comment:
this happened to me because the page types was misconfigured in the project. Maybe unnecessary?